### ISSUE TITLE
too few ticks

### ISSUE THREAD

**Issue by tpapp** [JuliaPlots/Plots.jl#1045]
While the intended number of ticks should fall between 5 and 8 ([cf](https://github.com/JuliaPlots/Plots.jl/blob/56bda82529676d15be417b7d92703964ce0ec2f8/src/axes.jl#L184)), sometimes that does not work. 

MWE:
```julia
using Plots
using StatPlots
gr() # important
x = linspace(0.2291313228541989, 0.3318750511155737, 100)
plot(density(x)) # 1 or 2 ticks
```
See [related discussion](https://discourse.julialang.org/t/more-ticks-in-plots-jl/5575).

**Comment by daschw** [comment]
(On master) You can specify the number of ticks you want to be shown by passing an `Int` to the ticks attribute.
E.g. `plot(density(x), xticks = 4)`
This does not always return the nicest results, though, because it allows the optimize_ticks algorithm to expand the axis limits. But you can try out some values or set the tick positions yourself via `plot(density(x), xticks = [0.2, 0.25, 0.3, 0.35])`

**Comment by mkborregaard** [comment]
As an aside, we've fixed the world age issue, so you no longer should have to call the backend explicitly (and no longer `using Plots` when you `using StatPlots`).

**Comment by piever** [comment]
Fixed by #1049 and [#26](https://github.com/JuliaPlots/PlotUtils.jl/pull/26)
